test/jest/mparticle-instance-manager.test.ts

```diff
@@ -0,0 +1,27 @@
+import Constants from '../../src/constants';
+import { IMParticleInstanceManager } from '../../src/sdkRuntimeModels';
+const mockPerformanceMark = (globalThis as any).__mockPerformanceMark;
+
+
+const mParticle = (globalThis as any).mParticle as IMParticleInstanceManager;
+
+describe('mParticle instance manager', () => {
+
+    it('sets sdkStart event timing', async () => {
+        expect(mockPerformanceMark).toHaveBeenCalledWith(Constants.PerformanceMetricsNames.SdkStart);
```

Comment:
i'm a bit confused as to how this test is passing.  mParticle is not being initiatlized at all, so how is the performance API being called?